### ISSUE TITLE
Cypress: A11Y validator race condition

### ISSUE THREAD

**Issue by pedropintosilva** [CollaboraOnline/online#15295]
make -C cypress_test run-desktop spec=calc/a11y_dialog_spec.js

* It passes locally, with core on commit baa003816fbca6cf37c1c7239fa1d3dcf721221d (origin/distro/collabora/co-26.04)
* it fails in CI

```js
                    Test failed: integration_tests/desktop/calc/a11y_dialog_spec.js / Accessibility Calc Dialog Tests / Shape paragraph dialog
                    
                    Found A11y errors:
                    A11yValidatorException: In 'Paragraph' at 'ParagraphDialog > dialog-vbox1 > tabcontrol > Alignment > ParaAlignPage > maingrid160 > frameJUSTIFICATION > frameJUSTIFICATION-content > gridJustification > labelWordSpacing': label element in widget of type 'labelWordSpacing' is associated with element with id: 'spin_WORD_SPACING_MAX-input' via htmlFor, but that element also has aria-label or aria-labelledby attribute. Should not duplicate
```

[#1452.txt](https://github.com/user-attachments/files/26371087/1452.txt)

cypress seems to fail in more than one PR on "A11yValidatorException: In 'Paragraph' at 'ParagraphDialog > dialog-vbox1 > tabcontrol > Alignment > ParaAlignPage > maingrid160 > frameJUSTIFICATION > frameJUSTIFICATION-content > gridJustification > labelWordSpacing': label element in widget of type 'labelWordSpacing' is associated with element with id: 'spin_WORD_SPACING_MAX-input' via htmlFor, but that element also has aria-label or aria-labelledby attribute. Should not duplicate labelling." from A11yValidator.ts:188

Example PRs: https://github.com/CollaboraOnline/online/pull/15292 or https://github.com/CollaboraOnline/online/pull/15010

**Comment by mohit-marathe** [comment]
This fails locally too. It started failing after https://gerrit.libreoffice.org/c/core/+/202924.

**Comment by pedropintosilva** [comment]
ah I still don't have that commit in my core, thanks.

5b857902f93d `add support for multiple a11y label-for`. So, the core added multi-label-for support, which caused labelWordSpacing to get labelFor pointing to spin_WORD_SPACING_MAX (even without mnemonic-widget).
